src/hssm/distribution_utils/dist.py

```diff
@@ -276,8 +332,24 @@ def dist(cls, **kwargs):  # pylint: disable=arguments-renamed
             return super().dist(dist_params, **other_kwargs)
 
         def logp(data, *dist_params):  # pylint: disable=E0213
+            if list_params[-1] == "p_outlier":
+                p_outlier = dist_params[-1]
+                dist_params = dist_params[:-1]
+
             logp = loglik(data, *dist_params)
 
+            if list_params[-1] == "p_outlier":
```

Comment:
just flagging, this should be bypassed if `p_outlier` was `None` but also if `p_outlier` was set to 0

Reply:
This is resolved through the `HSSM` class (when `p_outlier` is set to `None` or `0`, `lapse` passed to it will be `None`)